experiments/run_experiment.py

```diff
@@ -446,8 +562,34 @@ def run_batch(
     returned."""
     all_results = None
     for i, run_config in enumerate(run_configs):
-        paths, clusters = run_mrmc(**run_config)
-        run_results = format_results(paths, clusters, run_config)
+        if recourse_method == "mrmc":
```

Comment:
Worth making a note for later to rename all instances of MRMC to StEP in the codebase.

Reply:
Agreed. https://github.com/jakeval/MRMC/issues/56.